nutaudet/Target.cxx

```diff
@@ -650,7 +641,7 @@ Bool_t  Target::ProcessHits(FairVolume* vol)
 
     Int_t motherV[MaxL];
 //   Bool_t EmTop = 0, EmBot = 0, EmCESTop = 0, EmCESBot = 0;
-    Bool_t EmBrick = 0, EmCES = 0, EmTop;
+    Bool_t EmBrick = 0, EmTop;
```

Comment:
Please don't define multiple things on the same line, this is unreadable.
Also, `Bool_t` should be initialised to `false`, not `0` for readability.

Reply:
Ok, done.